src/skopeo_service.go

```
@@ -150,34 +159,133 @@ func handleWebSocket(c *gin.Context) {
 	})
+	
+	// 启动读取循环（主要用于处理pong和检测断开）
+	go client.readPump()
+}
+
+// 启动心跳保活机制
+func (c *Client) startHeartbeat() {
+	c.heartbeat = time.NewTicker(30 * time.Second)
+	go func() {
+		defer c.heartbeat.Stop()
+		for {
```

golangci-lint (1.64.8)

S1000: should use for range instead of for { select {} }

(gosimple)